src/Bus/bus-command.pl

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use FindBin qw($RealBin);
+use lib "$RealBin/..";
+use lib "$RealBin/../deps";
```

Comment:
`use lib "$RealBin/../.."`, since XSTools.dll is there by default.

The same goes for src/Bus/bus-server.pl